service/routes.py

```diff
@@ -97,6 +97,35 @@ def list_shopcarts():
     app.logger.info("Returning %d shopcarts", len(results))
     return jsonify(results), status.HTTP_200_OK
 
+######################################################################
```

Comment:
From my point of view, update should delete the older one. Here you just added the new products. The new products could be the same as the one that's already in the shopcart. So it's better to just empty the shocpart first, and then add new product.